src/QsCompiler/QirGeneration/Context.cs

```diff
@@ -481,6 +481,17 @@ internal GlobalVariable GetOrCreateStringConstant(string str)
             return constant;
         }
 
+        /// <summary>
+        /// Creates a global constant (data array) that stores the given bytes.
+        /// </summary>
+        internal GlobalVariable CreateArrayConstant(byte[] array) {
```

Comment:
How about naming it CreateGlobalConstantArray, to be a bit more descriptive?

Reply:
Yes, this is a much better name! (I didn't like the previous name, just named it similar to the string function.)